vfs.go

```diff
@@ -643,3 +644,52 @@ func (v *VFS) List(path string) ([]string, []string, error) {
 
 	return folderData.Folders, folderData.Files, nil
 }
+
+type visitRecursiveCallback = func(path string, size uint64) (doContinue bool, err error)
```

Comment:
We might want to export this type because `VisitRecursive` may otherwise be annoying to use: Not sure if all IDEs display non-exported types. If an IDE doesn't, you have to look at TileDB-Go's source code to figure out what function signature is expected here.
```suggestion
type VisitRecursiveCallback = func(path string, size uint64) (doContinue bool, err error)
```

Reply:
I didn't want to copy-paste the callback's signature, but also didn't want to export a type without it being needed, but if some IDEs might not like it sure, done.